src/main/java/com/formulasearchengine/mathmltools/mml/CMMLInfo.java

```diff
@@ -139,15 +101,11 @@ private void abstractNodeCD(Node node) {
         String cd;
         try {
             cd = node.getAttributes().getNamedItem("cd").getNodeValue();
-        } catch (final DOMException e) {
+        } catch (final DOMException | NullPointerException e) {
             //TODO: Implement CD fallback
```

Comment:
if we had a logger, (do we?) I would like to log theses kind of exceptions at least on a debug log level.